R/pairwise-comparisons.R

```diff
@@ -405,10 +443,15 @@ infer_rel_skill_metric <- function(scores) {
   } else if ("brier_score" %in% colnames(scores)) {
     rel_skill_metric <- "brier_score"
   } else {
-    stop(
-      "automatically assigning a metric to compute relative skills on failed. ",
-      "Please provide a metric."
+    #nolint start: keyword_quote_linter
+    cli_abort(
+      c(
+        "Automatically assigning a metric to compute relative skills
```

Comment:
```suggestion
        "!" = "Automatically assigning a metric to compute relative skills
```

(I think that function will be dropped anyway soon see #673